Build failure: dmd

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

```
nix build '.#dmd'
```

### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

Couldn't find a relevant build job

### Relevant log output

```console
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/0da9gqpxrii8na3jk7hxzmj6h4kzaf9i-dmd
unpacking source archive /nix/store/6f7wfi95zhjrj8g4b3fz5l8jdln4yp4g-phobos
source root is .
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
applying patch /nix/store/j5wza4a87rdnwrrqm5pgx0fcf0vn5n85-fdd25893e0ac04893d6eba8652903d499b7b0dfc.patch
patching file dmd/compiler/test/fail_compilation/needspkgmod.d
patching script interpreter paths in dmd/compiler/test/runnable/cdvecfill.sh dmd/compiler/test/runnable/gdb15729.sh dmd/compiler/test/runnable/importc-test1.sh dmd/compiler/test/runnable/link14198a.sh dmd/compiler/test/runnable/link14834.sh dmd/compiler/test/runnable/link20802.sh dmd/compiler/test/runnable/link846.sh dmd/compiler/test/runnable/test10386.sh dmd/compiler/test/runnable/test10567.sh dmd/compiler/test/runnable/test13666.sh dmd/compiler/test/runnable/test13742.sh dmd/compiler/test/runnable/test13774.sh dmd/compiler/test/runnable/test16096.sh dmd/compiler/test/runnable/test17619.sh dmd/compiler/test/runnable/test18076.sh dmd/compiler/test/runnable/test18141.sh dmd/compiler/test/runnable/test18335.sh dmd/compiler/test/runnable/test18412.sh dmd/compiler/test/runnable/test18456.sh dmd/compiler/test/runnable/test18902.sh dmd/compiler/test/runnable/test21234.sh dmd/compiler/test/runnable/test21723.sh dmd/compiler/test/runnable/test22863.sh dmd/compiler/test/runnable/test23148.sh dmd/compiler/test/runnable/test24590.sh dmd/compiler/test/runnable/test35.sh dmd/compiler/test/runnable/test39.sh dmd/compiler/test/runnable/test9287.sh dmd/compiler/test/runnable/test_safe_final_switch.sh dmd/compiler/test/runnable/test_switches.sh dmd/compiler/test/runnable/extra-files/coverage-postscript.sh dmd/compiler/test/runnable/extra-files/hello-profile-postscript.sh dmd/compiler/test/fail_compilation/fail12485.sh dmd/compiler/test/fail_compilation/fail6592.sh dmd/compiler/test/compilable/ddoc9764.sh dmd/compiler/test/compilable/dmdcliflags.sh dmd/compiler/test/compilable/issue15574.sh dmd/compiler/test/compilable/issue17167.sh dmd/compiler/test/compilable/issue19243.sh dmd/compiler/test/compilable/issue19724.sh dmd/compiler/test/compilable/json_nosource.sh dmd/compiler/test/compilable/test14894.sh dmd/compiler/test/compilable/test18367.sh dmd/compiler/test/compilable/test19266.sh dmd/compiler/test/compilable/test19376.sh dmd/compiler/test/compilable/test19463.sh dmd/compiler/test/compilable/test24171.sh dmd/compiler/test/compilable/test6461.sh dmd/compiler/test/compilable/test9680.sh dmd/compiler/test/compilable/testclidflags.sh dmd/compiler/test/compilable/testcolor.sh dmd/compiler/test/compilable/extra-files/ddocAny-postscript.sh dmd/compiler/test/compilable/extra-files/ddocYear-postscript.sh dmd/compiler/test/compilable/extra-files/json-postscript.sh dmd/compiler/test/compilable/extra-files/objdump-postscript.sh dmd/compiler/test/compilable/extra-files/vcg-ast-arraylength-postscript.sh dmd/compiler/test/tools/common_funcs.sh dmd/compiler/test/tools/exported_vars.sh dmd/compiler/test/tools/postscript.sh dmd/compiler/test/tools/sh_do_test.sh
dmd/compiler/test/runnable/gdb15729.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/link14198a.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/link14834.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/link20802.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/link846.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test10386.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test10567.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test13666.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test13742.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test13774.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test16096.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test17619.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test18076.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test18141.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test18335.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test18412.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test18902.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test21234.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test24590.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test35.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test39.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test9287.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test_safe_final_switch.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/test_switches.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/extra-files/coverage-postscript.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/runnable/extra-files/hello-profile-postscript.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/fail_compilation/fail12485.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/fail_compilation/fail6592.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/ddoc9764.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/issue17167.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/issue19724.sh: interpreter directive changed from "#! /usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/json_nosource.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/test14894.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/test18367.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/test19376.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/test19463.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/test24171.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/test6461.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/test9680.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/testclidflags.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/testcolor.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/extra-files/ddocAny-postscript.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/extra-files/ddocYear-postscript.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/extra-files/json-postscript.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/compilable/extra-files/vcg-ast-arraylength-postscript.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/tools/postscript.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
dmd/compiler/test/tools/sh_do_test.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash"
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
(DC) COMMON
(TX) DMD_CONF
(TX) SYSCONFDIR
(TX) VERSION
(DC) LEXER
(DC) BACKEND
(DC) DMD
Success
make: Entering directory '/build/dmd/druntime'
make: git: No such file or directory
/build/dmd/generated/linux/release/64/dmd -c -conf= -Isrc -Iimport -w -de -preview=dip1000 -preview=fieldwise -m64 -fPIC  -preview=dtorfields -O -release -inline  -version=Shared -fPIC -version=Shared -fPIC -I. -P=-I. src/core/stdc/errno.c -of../generated/linux/release/64/errno_c.o
gcc -c -m64 -fPIC -DHAVE_UNISTD_H -O3 src/core/threadasm.S -o../generated/linux/release/64/threadasm.o
gcc -c -m64 -fPIC -DHAVE_UNISTD_H -O3 src/etc/valgrind/valgrind.c -o../generated/linux/release/64/valgrind.o
/build/dmd/generated/linux/release/64/dmd -c -fPIC -of../generated/linux/release/64/libdruntime.so.o -conf= -Isrc -Iimport -w -de -preview=dip1000 -preview=fieldwise -m64 -fPIC  -preview=dtorfields -O -release -inline  -version=Shared -fPIC -version=Shared -fPIC src/object.d src/core/atomic.d src/core/attribute.d src/core/bitop.d src/core/builtins.d src/core/checkedint.d src/core/cpuid.d src/core/demangle.d src/core/exception.d src/core/factory.d src/core/int128.d src/core/interpolation.d src/core/lifetime.d src/core/math.d src/core/memory.d src/core/runtime.d src/core/simd.d src/core/time.d src/core/vararg.d src/core/volatile.d src/core/gc/config.d src/core/gc/gcinterface.d src/core/gc/registry.d src/core/internal/abort.d src/core/internal/atomic.d src/core/internal/attributes.d src/core/internal/convert.d src/core/internal/dassert.d src/core/internal/destruction.d src/core/internal/entrypoint.d src/core/internal/execinfo.d src/core/internal/hash.d src/core/internal/moving.d src/core/internal/newaa.d src/core/internal/parseoptions.d src/core/internal/postblit.d src/core/internal/qsort.d src/core/internal/spinlock.d src/core/internal/string.d src/core/internal/switch_.d src/core/internal/traits.d src/core/internal/utf.d src/core/internal/lifetime.d src/core/internal/array/appending.d src/core/internal/array/arrayassign.d src/core/internal/array/comparison.d src/core/internal/array/construction.d src/core/internal/array/equality.d src/core/internal/array/casting.d src/core/internal/array/capacity.d src/core/internal/array/concatenation.d src/core/internal/array/duplication.d src/core/internal/array/operations.d src/core/internal/array/utils.d src/core/internal/backtrace/dwarf.d src/core/internal/backtrace/elf.d src/core/internal/backtrace/handler.d src/core/internal/backtrace/libunwind.d src/core/internal/backtrace/macho.d src/core/internal/backtrace/unwind.d src/core/internal/container/array.d src/core/internal/container/common.d src/core/internal/container/hashtab.d src/core/internal/container/treap.d src/core/internal/elf/dl.d src/core/internal/elf/io.d src/core/internal/gc/bits.d src/core/internal/gc/os.d src/core/internal/gc/pooltable.d src/core/internal/gc/proxy.d src/core/internal/gc/impl/conservative/gc.d src/core/internal/gc/impl/manual/gc.d src/core/internal/gc/impl/proto/gc.d src/core/internal/util/array.d src/core/internal/util/math.d src/core/internal/vararg/aarch64.d src/core/internal/vararg/sysv_x64.d src/core/stdc/assert_.d src/core/stdc/stdatomic.d src/core/stdc/complex.d src/core/stdc/config.d src/core/stdc/ctype.d src/core/stdc/errno.d src/core/stdc/fenv.d src/core/stdc/float_.d src/core/stdc/inttypes.d src/core/stdc/limits.d src/core/stdc/locale.d src/core/stdc/math.d src/core/stdc/signal.d src/core/stdc/stdarg.d src/core/stdc/stddef.d src/core/stdc/stdint.d src/core/stdc/stdio.d src/core/stdc/stdlib.d src/core/stdc/string.d src/core/stdc/tgmath.d src/core/stdc/time.d src/core/stdc/wchar_.d src/core/stdc/wctype.d src/core/stdcpp/allocator.d src/core/stdcpp/array.d src/core/stdcpp/exception.d src/core/stdcpp/memory.d src/core/stdcpp/new_.d src/core/stdcpp/string.d src/core/stdcpp/string_view.d src/core/stdcpp/typeinfo.d src/core/stdcpp/type_traits.d src/core/stdcpp/utility.d src/core/stdcpp/vector.d src/core/stdcpp/xutility.d src/core/sync/barrier.d src/core/sync/condition.d src/core/sync/config.d src/core/sync/exception.d src/core/sync/event.d src/core/sync/mutex.d src/core/sync/rwmutex.d src/core/sync/semaphore.d src/core/sys/bionic/err.d src/core/sys/bionic/fcntl.d src/core/sys/bionic/stdlib.d src/core/sys/bionic/string.d src/core/sys/bionic/unistd.d src/core/sys/darwin/crt_externs.d src/core/sys/darwin/dlfcn.d src/core/sys/darwin/err.d src/core/sys/darwin/execinfo.d src/core/sys/darwin/fcntl.d src/core/sys/darwin/ifaddrs.d src/core/sys/darwin/pthread.d src/core/sys/darwin/stdlib.d src/core/sys/darwin/string.d src/core/sys/darwin/mach/dyld.d src/core/sys/darwin/mach/getsect.d src/core/sys/darwin/mach/kern_return.d src/core/sys/darwin/mach/loader.d src/core/sys/darwin/mach/nlist.d src/core/sys/darwin/mach/port.d src/core/sys/darwin/mach/semaphore.d src/core/sys/darwin/mach/stab.d src/core/sys/darwin/mach/thread_act.d src/core/sys/darwin/netinet/in_.d src/core/sys/darwin/sys/cdefs.d src/core/sys/darwin/sys/event.d src/core/sys/darwin/sys/mman.d src/core/sys/darwin/sys/sysctl.d src/core/sys/freebsd/dlfcn.d src/core/sys/freebsd/err.d src/core/sys/freebsd/execinfo.d src/core/sys/freebsd/ifaddrs.d src/core/sys/freebsd/mqueue.d src/core/sys/freebsd/pthread_np.d src/core/sys/freebsd/stdlib.d src/core/sys/freebsd/string.d src/core/sys/freebsd/time.d src/core/sys/freebsd/unistd.d src/core/sys/freebsd/net/if_.d src/core/sys/freebsd/net/if_dl.d src/core/sys/freebsd/netinet/in_.d src/core/sys/freebsd/sys/_bitset.d src/core/sys/freebsd/sys/_cpuset.d src/core/sys/freebsd/sys/cdefs.d src/core/sys/freebsd/sys/elf_common.d src/core/sys/freebsd/sys/elf.d src/core/sys/freebsd/sys/elf32.d src/core/sys/freebsd/sys/elf64.d src/core/sys/freebsd/sys/event.d src/core/sys/freebsd/sys/link_elf.d src/core/sys/freebsd/sys/mman.d src/core/sys/freebsd/sys/mount.d src/core/sys/freebsd/sys/socket.d src/core/sys/freebsd/sys/sysctl.d src/core/sys/freebsd/sys/types.d src/core/sys/dragonflybsd/dlfcn.d src/core/sys/dragonflybsd/err.d src/core/sys/dragonflybsd/execinfo.d src/core/sys/dragonflybsd/netinet/in_.d src/core/sys/dragonflybsd/pthread_np.d src/core/sys/dragonflybsd/stdlib.d src/core/sys/dragonflybsd/string.d src/core/sys/dragonflybsd/time.d src/core/sys/dragonflybsd/sys/_bitset.d src/core/sys/dragonflybsd/sys/_cpuset.d src/core/sys/dragonflybsd/sys/cdefs.d src/core/sys/dragonflybsd/sys/elf.d src/core/sys/dragonflybsd/sys/elf32.d src/core/sys/dragonflybsd/sys/elf64.d src/core/sys/dragonflybsd/sys/elf_common.d src/core/sys/dragonflybsd/sys/event.d src/core/sys/dragonflybsd/sys/link_elf.d src/core/sys/dragonflybsd/sys/mman.d src/core/sys/dragonflybsd/sys/socket.d src/core/sys/dragonflybsd/sys/sysctl.d src/core/sys/elf/package.d src/core/sys/linux/config.d src/core/sys/linux/dlfcn.d src/core/sys/linux/elf.d src/core/sys/linux/epoll.d src/core/sys/linux/err.d src/core/sys/linux/errno.d src/core/sys/linux/execinfo.d src/core/sys/linux/fcntl.d src/core/sys/linux/fs.d src/core/sys/linux/ifaddrs.d src/core/sys/linux/io_uring.d src/core/sys/linux/link.d src/core/sys/linux/perf_event.d src/core/sys/linux/sched.d src/core/sys/linux/stdio.d src/core/sys/linux/string.d src/core/sys/linux/termios.d src/core/sys/linux/time.d src/core/sys/linux/timerfd.d src/core/sys/linux/tipc.d src/core/sys/linux/unistd.d src/core/sys/linux/linux/if_arp.d src/core/sys/linux/linux/if_packet.d src/core/sys/linux/netinet/in_.d src/core/sys/linux/netinet/tcp.d src/core/sys/linux/sys/auxv.d src/core/sys/linux/sys/eventfd.d src/core/sys/linux/sys/file.d src/core/sys/linux/sys/inotify.d src/core/sys/linux/sys/mman.d src/core/sys/linux/sys/mount.d src/core/sys/linux/sys/signalfd.d src/core/sys/linux/sys/socket.d src/core/sys/linux/sys/sysinfo.d src/core/sys/linux/sys/xattr.d src/core/sys/linux/sys/time.d src/core/sys/linux/sys/prctl.d src/core/sys/netbsd/dlfcn.d src/core/sys/netbsd/err.d src/core/sys/netbsd/execinfo.d src/core/sys/netbsd/stdlib.d src/core/sys/netbsd/string.d src/core/sys/netbsd/time.d src/core/sys/netbsd/sys/elf.d src/core/sys/netbsd/sys/elf32.d src/core/sys/netbsd/sys/elf64.d src/core/sys/netbsd/sys/elf_common.d src/core/sys/netbsd/sys/event.d src/core/sys/netbsd/sys/featuretest.d src/core/sys/netbsd/sys/link_elf.d src/core/sys/netbsd/sys/mman.d src/core/sys/netbsd/sys/sysctl.d src/core/sys/openbsd/dlfcn.d src/core/sys/openbsd/err.d src/core/sys/openbsd/execinfo.d src/core/sys/openbsd/pthread_np.d src/core/sys/openbsd/pwd.d src/core/sys/openbsd/stdlib.d src/core/sys/openbsd/string.d src/core/sys/openbsd/time.d src/core/sys/openbsd/unistd.d src/core/sys/openbsd/sys/cdefs.d src/core/sys/openbsd/sys/elf.d src/core/sys/openbsd/sys/elf32.d src/core/sys/openbsd/sys/elf64.d src/core/sys/openbsd/sys/elf_common.d src/core/sys/openbsd/sys/link_elf.d src/core/sys/openbsd/sys/mman.d src/core/sys/openbsd/sys/sysctl.d src/core/sys/posix/arpa/inet.d src/core/sys/posix/aio.d src/core/sys/posix/config.d src/core/sys/posix/dirent.d src/core/sys/posix/dlfcn.d src/core/sys/posix/fcntl.d src/core/sys/posix/grp.d src/core/sys/posix/iconv.d src/core/sys/posix/inttypes.d src/core/sys/posix/libgen.d src/core/sys/posix/locale.d src/core/sys/posix/mqueue.d src/core/sys/posix/netdb.d src/core/sys/posix/poll.d src/core/sys/posix/pthread.d src/core/sys/posix/pwd.d src/core/sys/posix/sched.d src/core/sys/posix/semaphore.d src/core/sys/posix/setjmp.d src/core/sys/posix/signal.d src/core/sys/posix/spawn.d src/core/sys/posix/stdio.d src/core/sys/posix/stdlib.d src/core/sys/posix/string.d src/core/sys/posix/strings.d src/core/sys/posix/syslog.d src/core/sys/posix/termios.d src/core/sys/posix/time.d src/core/sys/posix/ucontext.d src/core/sys/posix/unistd.d src/core/sys/posix/utime.d src/core/sys/posix/net/if_.d src/core/sys/posix/netinet/in_.d src/core/sys/posix/netinet/tcp.d src/core/sys/posix/stdc/time.d src/core/sys/posix/sys/filio.d src/core/sys/posix/sys/ioccom.d src/core/sys/posix/sys/ioctl.d src/core/sys/posix/sys/ipc.d src/core/sys/posix/sys/mman.d src/core/sys/posix/sys/msg.d src/core/sys/posix/sys/resource.d src/core/sys/posix/sys/select.d src/core/sys/posix/sys/shm.d src/core/sys/posix/sys/socket.d src/core/sys/posix/sys/stat.d src/core/sys/posix/sys/statvfs.d src/core/sys/posix/sys/time.d src/core/sys/posix/sys/ttycom.d src/core/sys/posix/sys/types.d src/core/sys/posix/sys/uio.d src/core/sys/posix/sys/un.d src/core/sys/posix/sys/utsname.d src/core/sys/posix/sys/wait.d src/core/sys/solaris/dlfcn.d src/core/sys/solaris/elf.d src/core/sys/solaris/err.d src/core/sys/solaris/execinfo.d src/core/sys/solaris/libelf.d src/core/sys/solaris/link.d src/core/sys/solaris/stdlib.d src/core/sys/solaris/time.d src/core/sys/solaris/sys/elf.d src/core/sys/solaris/sys/elf_386.d src/core/sys/solaris/sys/elf_amd64.d src/core/sys/solaris/sys/elf_notes.d src/core/sys/solaris/sys/elf_SPARC.d src/core/sys/solaris/sys/elftypes.d src/core/sys/solaris/sys/link.d src/core/sys/solaris/sys/priocntl.d src/core/sys/solaris/sys/procset.d src/core/sys/solaris/sys/types.d src/core/sys/windows/accctrl.d src/core/sys/windows/aclapi.d src/core/sys/windows/aclui.d src/core/sys/windows/basetsd.d src/core/sys/windows/basetyps.d src/core/sys/windows/cderr.d src/core/sys/windows/cguid.d src/core/sys/windows/com.d src/core/sys/windows/comcat.d src/core/sys/windows/commctrl.d src/core/sys/windows/commdlg.d src/core/sys/windows/core.d src/core/sys/windows/cpl.d src/core/sys/windows/cplext.d src/core/sys/windows/custcntl.d src/core/sys/windows/dbghelp.d src/core/sys/windows/dbghelp_types.d src/core/sys/windows/dbt.d src/core/sys/windows/dde.d src/core/sys/windows/ddeml.d src/core/sys/windows/dhcpcsdk.d src/core/sys/windows/dlgs.d src/core/sys/windows/dll.d src/core/sys/windows/docobj.d src/core/sys/windows/errorrep.d src/core/sys/windows/exdisp.d src/core/sys/windows/exdispid.d src/core/sys/windows/httpext.d src/core/sys/windows/idispids.d src/core/sys/windows/imagehlp.d src/core/sys/windows/imm.d src/core/sys/windows/intshcut.d src/core/sys/windows/ipexport.d src/core/sys/windows/iphlpapi.d src/core/sys/windows/ipifcons.d src/core/sys/windows/iprtrmib.d src/core/sys/windows/iptypes.d src/core/sys/windows/isguids.d src/core/sys/windows/lm.d src/core/sys/windows/lmaccess.d src/core/sys/windows/lmalert.d src/core/sys/windows/lmapibuf.d src/core/sys/windows/lmat.d src/core/sys/windows/lmaudit.d src/core/sys/windows/lmbrowsr.d src/core/sys/windows/lmchdev.d src/core/sys/windows/lmconfig.d src/core/sys/windows/lmcons.d src/core/sys/windows/lmerr.d src/core/sys/windows/lmerrlog.d src/core/sys/windows/lmmsg.d src/core/sys/windows/lmremutl.d src/core/sys/windows/lmrepl.d src/core/sys/windows/lmserver.d src/core/sys/windows/lmshare.d src/core/sys/windows/lmsname.d src/core/sys/windows/lmstats.d src/core/sys/windows/lmsvc.d src/core/sys/windows/lmuse.d src/core/sys/windows/lmuseflg.d src/core/sys/windows/lmwksta.d src/core/sys/windows/lzexpand.d src/core/sys/windows/mapi.d src/core/sys/windows/mciavi.d src/core/sys/windows/mcx.d src/core/sys/windows/mgmtapi.d src/core/sys/windows/mmsystem.d src/core/sys/windows/msacm.d src/core/sys/windows/mshtml.d src/core/sys/windows/mswsock.d src/core/sys/windows/nb30.d src/core/sys/windows/nddeapi.d src/core/sys/windows/nspapi.d src/core/sys/windows/ntdef.d src/core/sys/windows/ntdll.d src/core/sys/windows/ntldap.d src/core/sys/windows/ntsecapi.d src/core/sys/windows/ntsecpkg.d src/core/sys/windows/oaidl.d src/core/sys/windows/objbase.d src/core/sys/windows/objfwd.d src/core/sys/windows/objidl.d src/core/sys/windows/objsafe.d src/core/sys/windows/ocidl.d src/core/sys/windows/odbcinst.d src/core/sys/windows/ole.d src/core/sys/windows/ole2.d src/core/sys/windows/ole2ver.d src/core/sys/windows/oleacc.d src/core/sys/windows/oleauto.d src/core/sys/windows/olectl.d src/core/sys/windows/olectlid.d src/core/sys/windows/oledlg.d src/core/sys/windows/oleidl.d src/core/sys/windows/pbt.d src/core/sys/windows/powrprof.d src/core/sys/windows/prsht.d src/core/sys/windows/psapi.d src/core/sys/windows/rapi.d src/core/sys/windows/ras.d src/core/sys/windows/rasdlg.d src/core/sys/windows/raserror.d src/core/sys/windows/rassapi.d src/core/sys/windows/reason.d src/core/sys/windows/regstr.d src/core/sys/windows/richedit.d src/core/sys/windows/richole.d src/core/sys/windows/rpc.d src/core/sys/windows/rpcdce.d src/core/sys/windows/rpcdce2.d src/core/sys/windows/rpcdcep.d src/core/sys/windows/rpcndr.d src/core/sys/windows/rpcnsi.d src/core/sys/windows/rpcnsip.d src/core/sys/windows/rpcnterr.d src/core/sys/windows/schannel.d src/core/sys/windows/sdkddkver.d src/core/sys/windows/secext.d src/core/sys/windows/security.d src/core/sys/windows/servprov.d src/core/sys/windows/setupapi.d src/core/sys/windows/shellapi.d src/core/sys/windows/shldisp.d src/core/sys/windows/shlguid.d src/core/sys/windows/shlobj.d src/core/sys/windows/shlwapi.d src/core/sys/windows/snmp.d src/core/sys/windows/sql.d src/core/sys/windows/sqlext.d src/core/sys/windows/sqltypes.d src/core/sys/windows/sqlucode.d src/core/sys/windows/sspi.d src/core/sys/windows/stacktrace.d src/core/sys/windows/stat.d src/core/sys/windows/stdc/malloc.d src/core/sys/windows/stdc/time.d src/core/sys/windows/subauth.d src/core/sys/windows/threadaux.d src/core/sys/windows/tlhelp32.d src/core/sys/windows/tmschema.d src/core/sys/windows/unknwn.d src/core/sys/windows/uuid.d src/core/sys/windows/vfw.d src/core/sys/windows/w32api.d src/core/sys/windows/winbase.d src/core/sys/windows/winber.d src/core/sys/windows/wincon.d src/core/sys/windows/wincrypt.d src/core/sys/windows/windef.d src/core/sys/windows/windows.d src/core/sys/windows/winerror.d src/core/sys/windows/wingdi.d src/core/sys/windows/winhttp.d src/core/sys/windows/wininet.d src/core/sys/windows/winioctl.d src/core/sys/windows/winldap.d src/core/sys/windows/winnetwk.d src/core/sys/windows/winnls.d src/core/sys/windows/winnt.d src/core/sys/windows/winperf.d src/core/sys/windows/winreg.d src/core/sys/windows/winsock2.d src/core/sys/windows/winspool.d src/core/sys/windows/winsvc.d src/core/sys/windows/winuser.d src/core/sys/windows/winver.d src/core/sys/windows/wtsapi32.d src/core/sys/windows/wtypes.d src/core/thread/fiber.d src/core/thread/types.d src/core/thread/threadgroup.d src/core/thread/threadbase.d src/core/thread/osthread.d src/core/thread/context.d src/core/thread/package.d src/rt/aApply.d src/rt/aApplyR.d src/rt/aaA.d src/rt/adi.d src/rt/alloca.d src/rt/arraycat.d src/rt/cast_.d src/rt/cmath2.d src/rt/config.d src/rt/cover.d src/rt/critical_.d src/rt/deh.d src/rt/deh_win32.d src/rt/deh_win64_posix.d src/rt/dmain2.d src/rt/dwarfeh.d src/rt/ehalloc.d src/rt/invariant.d src/rt/lifetime.d src/rt/llmath.d src/rt/memory.d src/rt/memset.d src/rt/minfo.d src/rt/monitor_.d src/rt/msvc.d src/rt/msvc_math.d src/rt/profilegc.d src/rt/sections.d src/rt/sections_darwin_64.d src/rt/sections_elf_shared.d src/rt/sections_osx_x86.d src/rt/sections_osx_x86_64.d src/rt/sections_solaris.d src/rt/sections_win64.d src/rt/tlsgc.d src/rt/trace.d src/rt/tracegc.d src/rt/util/typeinfo.d src/rt/util/utility.d src/etc/valgrind/valgrind.d src/etc/linux/memoryerror.d
/build/dmd/generated/linux/release/64/dmd -lib -of../generated/linux/release/64/libdruntime.a -Xfdruntime.json -conf= -Isrc -Iimport -w -de -preview=dip1000 -preview=fieldwise -m64 -fPIC  -preview=dtorfields -O -release -inline  src/object.d src/core/atomic.d src/core/attribute.d src/core/bitop.d src/core/builtins.d src/core/checkedint.d src/core/cpuid.d src/core/demangle.d src/core/exception.d src/core/factory.d src/core/int128.d src/core/interpolation.d src/core/lifetime.d src/core/math.d src/core/memory.d src/core/runtime.d src/core/simd.d src/core/time.d src/core/vararg.d src/core/volatile.d src/core/gc/config.d src/core/gc/gcinterface.d src/core/gc/registry.d src/core/internal/abort.d src/core/internal/atomic.d src/core/internal/attributes.d src/core/internal/convert.d src/core/internal/dassert.d src/core/internal/destruction.d src/core/internal/entrypoint.d src/core/internal/execinfo.d src/core/internal/hash.d src/core/internal/moving.d src/core/internal/newaa.d src/core/internal/parseoptions.d src/core/internal/postblit.d src/core/internal/qsort.d src/core/internal/spinlock.d src/core/internal/string.d src/core/internal/switch_.d src/core/internal/traits.d src/core/internal/utf.d src/core/internal/lifetime.d src/core/internal/array/appending.d src/core/internal/array/arrayassign.d src/core/internal/array/comparison.d src/core/internal/array/construction.d src/core/internal/array/equality.d src/core/internal/array/casting.d src/core/internal/array/capacity.d src/core/internal/array/concatenation.d src/core/internal/array/duplication.d src/core/internal/array/operations.d src/core/internal/array/utils.d src/core/internal/backtrace/dwarf.d src/core/internal/backtrace/elf.d src/core/internal/backtrace/handler.d src/core/internal/backtrace/libunwind.d src/core/internal/backtrace/macho.d src/core/internal/backtrace/unwind.d src/core/internal/container/array.d src/core/internal/container/common.d src/core/internal/container/hashtab.d src/core/internal/container/treap.d src/core/internal/elf/dl.d src/core/internal/elf/io.d src/core/internal/gc/bits.d src/core/internal/gc/os.d src/core/internal/gc/pooltable.d src/core/internal/gc/proxy.d src/core/internal/gc/impl/conservative/gc.d src/core/internal/gc/impl/manual/gc.d src/core/internal/gc/impl/proto/gc.d src/core/internal/util/array.d src/core/internal/util/math.d src/core/internal/vararg/aarch64.d src/core/internal/vararg/sysv_x64.d src/core/stdc/assert_.d src/core/stdc/stdatomic.d src/core/stdc/complex.d src/core/stdc/config.d src/core/stdc/ctype.d src/core/stdc/errno.d src/core/stdc/fenv.d src/core/stdc/float_.d src/core/stdc/inttypes.d src/core/stdc/limits.d src/core/stdc/locale.d src/core/stdc/math.d src/core/stdc/signal.d src/core/stdc/stdarg.d src/core/stdc/stddef.d src/core/stdc/stdint.d src/core/stdc/stdio.d src/core/stdc/stdlib.d src/core/stdc/string.d src/core/stdc/tgmath.d src/core/stdc/time.d src/core/stdc/wchar_.d src/core/stdc/wctype.d src/core/stdcpp/allocator.d src/core/stdcpp/array.d src/core/stdcpp/exception.d src/core/stdcpp/memory.d src/core/stdcpp/new_.d src/core/stdcpp/string.d src/core/stdcpp/string_view.d src/core/stdcpp/typeinfo.d src/core/stdcpp/type_traits.d src/core/stdcpp/utility.d src/core/stdcpp/vector.d src/core/stdcpp/xutility.d src/core/sync/barrier.d src/core/sync/condition.d src/core/sync/config.d src/core/sync/exception.d src/core/sync/event.d src/core/sync/mutex.d src/core/sync/rwmutex.d src/core/sync/semaphore.d src/core/sys/bionic/err.d src/core/sys/bionic/fcntl.d src/core/sys/bionic/stdlib.d src/core/sys/bionic/string.d src/core/sys/bionic/unistd.d src/core/sys/darwin/crt_externs.d src/core/sys/darwin/dlfcn.d src/core/sys/darwin/err.d src/core/sys/darwin/execinfo.d src/core/sys/darwin/fcntl.d src/core/sys/darwin/ifaddrs.d src/core/sys/darwin/pthread.d src/core/sys/darwin/stdlib.d src/core/sys/darwin/string.d src/core/sys/darwin/mach/dyld.d src/core/sys/darwin/mach/getsect.d src/core/sys/darwin/mach/kern_return.d src/core/sys/darwin/mach/loader.d src/core/sys/darwin/mach/nlist.d src/core/sys/darwin/mach/port.d src/core/sys/darwin/mach/semaphore.d src/core/sys/darwin/mach/stab.d src/core/sys/darwin/mach/thread_act.d src/core/sys/darwin/netinet/in_.d src/core/sys/darwin/sys/cdefs.d src/core/sys/darwin/sys/event.d src/core/sys/darwin/sys/mman.d src/core/sys/darwin/sys/sysctl.d src/core/sys/freebsd/dlfcn.d src/core/sys/freebsd/err.d src/core/sys/freebsd/execinfo.d src/core/sys/freebsd/ifaddrs.d src/core/sys/freebsd/mqueue.d src/core/sys/freebsd/pthread_np.d src/core/sys/freebsd/stdlib.d src/core/sys/freebsd/string.d src/core/sys/freebsd/time.d src/core/sys/freebsd/unistd.d src/core/sys/freebsd/net/if_.d src/core/sys/freebsd/net/if_dl.d src/core/sys/freebsd/netinet/in_.d src/core/sys/freebsd/sys/_bitset.d src/core/sys/freebsd/sys/_cpuset.d src/core/sys/freebsd/sys/cdefs.d src/core/sys/freebsd/sys/elf_common.d src/core/sys/freebsd/sys/elf.d src/core/sys/freebsd/sys/elf32.d src/core/sys/freebsd/sys/elf64.d src/core/sys/freebsd/sys/event.d src/core/sys/freebsd/sys/link_elf.d src/core/sys/freebsd/sys/mman.d src/core/sys/freebsd/sys/mount.d src/core/sys/freebsd/sys/socket.d src/core/sys/freebsd/sys/sysctl.d src/core/sys/freebsd/sys/types.d src/core/sys/dragonflybsd/dlfcn.d src/core/sys/dragonflybsd/err.d src/core/sys/dragonflybsd/execinfo.d src/core/sys/dragonflybsd/netinet/in_.d src/core/sys/dragonflybsd/pthread_np.d src/core/sys/dragonflybsd/stdlib.d src/core/sys/dragonflybsd/string.d src/core/sys/dragonflybsd/time.d src/core/sys/dragonflybsd/sys/_bitset.d src/core/sys/dragonflybsd/sys/_cpuset.d src/core/sys/dragonflybsd/sys/cdefs.d src/core/sys/dragonflybsd/sys/elf.d src/core/sys/dragonflybsd/sys/elf32.d src/core/sys/dragonflybsd/sys/elf64.d src/core/sys/dragonflybsd/sys/elf_common.d src/core/sys/dragonflybsd/sys/event.d src/core/sys/dragonflybsd/sys/link_elf.d src/core/sys/dragonflybsd/sys/mman.d src/core/sys/dragonflybsd/sys/socket.d src/core/sys/dragonflybsd/sys/sysctl.d src/core/sys/elf/package.d src/core/sys/linux/config.d src/core/sys/linux/dlfcn.d src/core/sys/linux/elf.d src/core/sys/linux/epoll.d src/core/sys/linux/err.d src/core/sys/linux/errno.d src/core/sys/linux/execinfo.d src/core/sys/linux/fcntl.d src/core/sys/linux/fs.d src/core/sys/linux/ifaddrs.d src/core/sys/linux/io_uring.d src/core/sys/linux/link.d src/core/sys/linux/perf_event.d src/core/sys/linux/sched.d src/core/sys/linux/stdio.d src/core/sys/linux/string.d src/core/sys/linux/termios.d src/core/sys/linux/time.d src/core/sys/linux/timerfd.d src/core/sys/linux/tipc.d src/core/sys/linux/unistd.d src/core/sys/linux/linux/if_arp.d src/core/sys/linux/linux/if_packet.d src/core/sys/linux/netinet/in_.d src/core/sys/linux/netinet/tcp.d src/core/sys/linux/sys/auxv.d src/core/sys/linux/sys/eventfd.d src/core/sys/linux/sys/file.d src/core/sys/linux/sys/inotify.d src/core/sys/linux/sys/mman.d src/core/sys/linux/sys/mount.d src/core/sys/linux/sys/signalfd.d src/core/sys/linux/sys/socket.d src/core/sys/linux/sys/sysinfo.d src/core/sys/linux/sys/xattr.d src/core/sys/linux/sys/time.d src/core/sys/linux/sys/prctl.d src/core/sys/netbsd/dlfcn.d src/core/sys/netbsd/err.d src/core/sys/netbsd/execinfo.d src/core/sys/netbsd/stdlib.d src/core/sys/netbsd/string.d src/core/sys/netbsd/time.d src/core/sys/netbsd/sys/elf.d src/core/sys/netbsd/sys/elf32.d src/core/sys/netbsd/sys/elf64.d src/core/sys/netbsd/sys/elf_common.d src/core/sys/netbsd/sys/event.d src/core/sys/netbsd/sys/featuretest.d src/core/sys/netbsd/sys/link_elf.d src/core/sys/netbsd/sys/mman.d src/core/sys/netbsd/sys/sysctl.d src/core/sys/openbsd/dlfcn.d src/core/sys/openbsd/err.d src/core/sys/openbsd/execinfo.d src/core/sys/openbsd/pthread_np.d src/core/sys/openbsd/pwd.d src/core/sys/openbsd/stdlib.d src/core/sys/openbsd/string.d src/core/sys/openbsd/time.d src/core/sys/openbsd/unistd.d src/core/sys/openbsd/sys/cdefs.d src/core/sys/openbsd/sys/elf.d src/core/sys/openbsd/sys/elf32.d src/core/sys/openbsd/sys/elf64.d src/core/sys/openbsd/sys/elf_common.d src/core/sys/openbsd/sys/link_elf.d src/core/sys/openbsd/sys/mman.d src/core/sys/openbsd/sys/sysctl.d src/core/sys/posix/arpa/inet.d src/core/sys/posix/aio.d src/core/sys/posix/config.d src/core/sys/posix/dirent.d src/core/sys/posix/dlfcn.d src/core/sys/posix/fcntl.d src/core/sys/posix/grp.d src/core/sys/posix/iconv.d src/core/sys/posix/inttypes.d src/core/sys/posix/libgen.d src/core/sys/posix/locale.d src/core/sys/posix/mqueue.d src/core/sys/posix/netdb.d src/core/sys/posix/poll.d src/core/sys/posix/pthread.d src/core/sys/posix/pwd.d src/core/sys/posix/sched.d src/core/sys/posix/semaphore.d src/core/sys/posix/setjmp.d src/core/sys/posix/signal.d src/core/sys/posix/spawn.d src/core/sys/posix/stdio.d src/core/sys/posix/stdlib.d src/core/sys/posix/string.d src/core/sys/posix/strings.d src/core/sys/posix/syslog.d src/core/sys/posix/termios.d src/core/sys/posix/time.d src/core/sys/posix/ucontext.d src/core/sys/posix/unistd.d src/core/sys/posix/utime.d src/core/sys/posix/net/if_.d src/core/sys/posix/netinet/in_.d src/core/sys/posix/netinet/tcp.d src/core/sys/posix/stdc/time.d src/core/sys/posix/sys/filio.d src/core/sys/posix/sys/ioccom.d src/core/sys/posix/sys/ioctl.d src/core/sys/posix/sys/ipc.d src/core/sys/posix/sys/mman.d src/core/sys/posix/sys/msg.d src/core/sys/posix/sys/resource.d src/core/sys/posix/sys/select.d src/core/sys/posix/sys/shm.d src/core/sys/posix/sys/socket.d src/core/sys/posix/sys/stat.d src/core/sys/posix/sys/statvfs.d src/core/sys/posix/sys/time.d src/core/sys/posix/sys/ttycom.d src/core/sys/posix/sys/types.d src/core/sys/posix/sys/uio.d src/core/sys/posix/sys/un.d src/core/sys/posix/sys/utsname.d src/core/sys/posix/sys/wait.d src/core/sys/solaris/dlfcn.d src/core/sys/solaris/elf.d src/core/sys/solaris/err.d src/core/sys/solaris/execinfo.d src/core/sys/solaris/libelf.d src/core/sys/solaris/link.d src/core/sys/solaris/stdlib.d src/core/sys/solaris/time.d src/core/sys/solaris/sys/elf.d src/core/sys/solaris/sys/elf_386.d src/core/sys/solaris/sys/elf_amd64.d src/core/sys/solaris/sys/elf_notes.d src/core/sys/solaris/sys/elf_SPARC.d src/core/sys/solaris/sys/elftypes.d src/core/sys/solaris/sys/link.d src/core/sys/solaris/sys/priocntl.d src/core/sys/solaris/sys/procset.d src/core/sys/solaris/sys/types.d src/core/sys/windows/accctrl.d src/core/sys/windows/aclapi.d src/core/sys/windows/aclui.d src/core/sys/windows/basetsd.d src/core/sys/windows/basetyps.d src/core/sys/windows/cderr.d src/core/sys/windows/cguid.d src/core/sys/windows/com.d src/core/sys/windows/comcat.d src/core/sys/windows/commctrl.d src/core/sys/windows/commdlg.d src/core/sys/windows/core.d src/core/sys/windows/cpl.d src/core/sys/windows/cplext.d src/core/sys/windows/custcntl.d src/core/sys/windows/dbghelp.d src/core/sys/windows/dbghelp_types.d src/core/sys/windows/dbt.d src/core/sys/windows/dde.d src/core/sys/windows/ddeml.d src/core/sys/windows/dhcpcsdk.d src/core/sys/windows/dlgs.d src/core/sys/windows/dll.d src/core/sys/windows/docobj.d src/core/sys/windows/errorrep.d src/core/sys/windows/exdisp.d src/core/sys/windows/exdispid.d src/core/sys/windows/httpext.d src/core/sys/windows/idispids.d src/core/sys/windows/imagehlp.d src/core/sys/windows/imm.d src/core/sys/windows/intshcut.d src/core/sys/windows/ipexport.d src/core/sys/windows/iphlpapi.d src/core/sys/windows/ipifcons.d src/core/sys/windows/iprtrmib.d src/core/sys/windows/iptypes.d src/core/sys/windows/isguids.d src/core/sys/windows/lm.d src/core/sys/windows/lmaccess.d src/core/sys/windows/lmalert.d src/core/sys/windows/lmapibuf.d src/core/sys/windows/lmat.d src/core/sys/windows/lmaudit.d src/core/sys/windows/lmbrowsr.d src/core/sys/windows/lmchdev.d src/core/sys/windows/lmconfig.d src/core/sys/windows/lmcons.d src/core/sys/windows/lmerr.d src/core/sys/windows/lmerrlog.d src/core/sys/windows/lmmsg.d src/core/sys/windows/lmremutl.d src/core/sys/windows/lmrepl.d src/core/sys/windows/lmserver.d src/core/sys/windows/lmshare.d src/core/sys/windows/lmsname.d src/core/sys/windows/lmstats.d src/core/sys/windows/lmsvc.d src/core/sys/windows/lmuse.d src/core/sys/windows/lmuseflg.d src/core/sys/windows/lmwksta.d src/core/sys/windows/lzexpand.d src/core/sys/windows/mapi.d src/core/sys/windows/mciavi.d src/core/sys/windows/mcx.d src/core/sys/windows/mgmtapi.d src/core/sys/windows/mmsystem.d src/core/sys/windows/msacm.d src/core/sys/windows/mshtml.d src/core/sys/windows/mswsock.d src/core/sys/windows/nb30.d src/core/sys/windows/nddeapi.d src/core/sys/windows/nspapi.d src/core/sys/windows/ntdef.d src/core/sys/windows/ntdll.d src/core/sys/windows/ntldap.d src/core/sys/windows/ntsecapi.d src/core/sys/windows/ntsecpkg.d src/core/sys/windows/oaidl.d src/core/sys/windows/objbase.d src/core/sys/windows/objfwd.d src/core/sys/windows/objidl.d src/core/sys/windows/objsafe.d src/core/sys/windows/ocidl.d src/core/sys/windows/odbcinst.d src/core/sys/windows/ole.d src/core/sys/windows/ole2.d src/core/sys/windows/ole2ver.d src/core/sys/windows/oleacc.d src/core/sys/windows/oleauto.d src/core/sys/windows/olectl.d src/core/sys/windows/olectlid.d src/core/sys/windows/oledlg.d src/core/sys/windows/oleidl.d src/core/sys/windows/pbt.d src/core/sys/windows/powrprof.d src/core/sys/windows/prsht.d src/core/sys/windows/psapi.d src/core/sys/windows/rapi.d src/core/sys/windows/ras.d src/core/sys/windows/rasdlg.d src/core/sys/windows/raserror.d src/core/sys/windows/rassapi.d src/core/sys/windows/reason.d src/core/sys/windows/regstr.d src/core/sys/windows/richedit.d src/core/sys/windows/richole.d src/core/sys/windows/rpc.d src/core/sys/windows/rpcdce.d src/core/sys/windows/rpcdce2.d src/core/sys/windows/rpcdcep.d src/core/sys/windows/rpcndr.d src/core/sys/windows/rpcnsi.d src/core/sys/windows/rpcnsip.d src/core/sys/windows/rpcnterr.d src/core/sys/windows/schannel.d src/core/sys/windows/sdkddkver.d src/core/sys/windows/secext.d src/core/sys/windows/security.d src/core/sys/windows/servprov.d src/core/sys/windows/setupapi.d src/core/sys/windows/shellapi.d src/core/sys/windows/shldisp.d src/core/sys/windows/shlguid.d src/core/sys/windows/shlobj.d src/core/sys/windows/shlwapi.d src/core/sys/windows/snmp.d src/core/sys/windows/sql.d src/core/sys/windows/sqlext.d src/core/sys/windows/sqltypes.d src/core/sys/windows/sqlucode.d src/core/sys/windows/sspi.d src/core/sys/windows/stacktrace.d src/core/sys/windows/stat.d src/core/sys/windows/stdc/malloc.d src/core/sys/windows/stdc/time.d src/core/sys/windows/subauth.d src/core/sys/windows/threadaux.d src/core/sys/windows/tlhelp32.d src/core/sys/windows/tmschema.d src/core/sys/windows/unknwn.d src/core/sys/windows/uuid.d src/core/sys/windows/vfw.d src/core/sys/windows/w32api.d src/core/sys/windows/winbase.d src/core/sys/windows/winber.d src/core/sys/windows/wincon.d src/core/sys/windows/wincrypt.d src/core/sys/windows/windef.d src/core/sys/windows/windows.d src/core/sys/windows/winerror.d src/core/sys/windows/wingdi.d src/core/sys/windows/winhttp.d src/core/sys/windows/wininet.d src/core/sys/windows/winioctl.d src/core/sys/windows/winldap.d src/core/sys/windows/winnetwk.d src/core/sys/windows/winnls.d src/core/sys/windows/winnt.d src/core/sys/windows/winperf.d src/core/sys/windows/winreg.d src/core/sys/windows/winsock2.d src/core/sys/windows/winspool.d src/core/sys/windows/winsvc.d src/core/sys/windows/winuser.d src/core/sys/windows/winver.d src/core/sys/windows/wtsapi32.d src/core/sys/windows/wtypes.d src/core/thread/fiber.d src/core/thread/types.d src/core/thread/threadgroup.d src/core/thread/threadbase.d src/core/thread/osthread.d src/core/thread/context.d src/core/thread/package.d src/rt/aApply.d src/rt/aApplyR.d src/rt/aaA.d src/rt/adi.d src/rt/alloca.d src/rt/arraycat.d src/rt/cast_.d src/rt/cmath2.d src/rt/config.d src/rt/cover.d src/rt/critical_.d src/rt/deh.d src/rt/deh_win32.d src/rt/deh_win64_posix.d src/rt/dmain2.d src/rt/dwarfeh.d src/rt/ehalloc.d src/rt/invariant.d src/rt/lifetime.d src/rt/llmath.d src/rt/memory.d src/rt/memset.d src/rt/minfo.d src/rt/monitor_.d src/rt/msvc.d src/rt/msvc_math.d src/rt/profilegc.d src/rt/sections.d src/rt/sections_darwin_64.d src/rt/sections_elf_shared.d src/rt/sections_osx_x86.d src/rt/sections_osx_x86_64.d src/rt/sections_solaris.d src/rt/sections_win64.d src/rt/tlsgc.d src/rt/trace.d src/rt/tracegc.d src/rt/util/typeinfo.d src/rt/util/utility.d src/etc/valgrind/valgrind.d src/etc/linux/memoryerror.d ../generated/linux/release/64/errno_c.o ../generated/linux/release/64/threadasm.o ../generated/linux/release/64/valgrind.o
/build/dmd/generated/linux/release/64/dmd -conf= -lib -of../generated/linux/release/64/libdruntime.so.a ../generated/linux/release/64/libdruntime.so.o ../generated/linux/release/64/errno_c.o ../generated/linux/release/64/threadasm.o ../generated/linux/release/64/valgrind.o
make: Leaving directory '/build/dmd/druntime'
make: Entering directory '/build/phobos'
make -C ../dmd/druntime MODEL=64 DMD=/build/dmd/generated/linux/release/64/dmd OS=linux BUILD=release
make[1]: Entering directory '/build/dmd/druntime'
make[1]: git: No such file or directory
make[1]: Nothing to be done for 'target'.
make[1]: Leaving directory '/build/dmd/druntime'
/build/dmd/generated/linux/release/64/dmd -c -version=TZDatabaseDir -version=LibcurlPath -J/build -conf= -I../dmd/druntime/import  -w -de -preview=dip1000 -preview=dtorfields -preview=fieldwise -m64 -fPIC -O -release -P=-DHAVE_UNISTD_H -P=-Ietc/c/zlib -ofgenerated/linux/release/64/zlib.o etc/c/zlib/adler32.c etc/c/zlib/compress.c etc/c/zlib/crc32.c etc/c/zlib/deflate.c etc/c/zlib/gzclose.c etc/c/zlib/gzlib.c etc/c/zlib/gzread.c etc/c/zlib/gzwrite.c etc/c/zlib/infback.c etc/c/zlib/inffast.c etc/c/zlib/inflate.c etc/c/zlib/inftrees.c etc/c/zlib/trees.c etc/c/zlib/uncompr.c etc/c/zlib/zutil.c
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
[1m/nix/store/qarrb8yfby1yyypm32vabzgxgq3w41ma-gcc-15.2.0/lib/gcc/x86_64-unknown-linux-gnu/15.2.0/include/stddef.h(465): [1;31mError: [mundefined identifier `[0;36m[m[1mnullptr[0;36m[m`, did you mean alias `[0;36m[m[1mnullptr_t[0;36m[m`?
make: *** [Makefile:353: generated/linux/release/64/zlib.o] Error 1
make: Leaving directory '/build/phobos'
```

### Additional context

_No response_

### System metadata

fetching path input 'path:/nix/store/nv0f6mcdixfwhjqxnnrk3iwfvkgbrxjl-source'
 - system: `"x86_64-linux"`
 - host os: `Linux 6.18.0-zen1, NixOS, 26.05 (Yarara), 26.05.20251215.1306659`
 - multi-user?: `yes`
 - sandbox: `relaxed`
 - version: `nix-env (Lix, like Nix) 2.93.3
System type: x86_64-linux
Additional system types: aarch64-linux, i686-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/corin/.config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/corin/.nix-profile/etc/xdg/nix/nix.conf:/home/corin/.local/state/nix/profile/etc/xdg/nix/nix.conf:/home/corin/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/corin/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf:/nix/store/xvwf72iv76k61bf55wi7w30h2drd1j57-gnome-settings-daemon-49.1/etc/xdg/nix/nix.conf:/nix/store/xvwf72iv76k61bf55wi7w30h2drd1j57-gnome-settings-daemon-49.1/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/w7x0993gjpy31k5v5xklnqmd8q0br8h0-lix-2.93.3/share`
 - channels(corin): `""`
 - nixpkgs: `/nix/store/nv0f6mcdixfwhjqxnnrk3iwfvkgbrxjl-source`

### Notify maintainers

@lionello @dukc @jtbx 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+build+failure%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/dm/dmd/generic.nix
+++ b/pkgs/by-name/dm/dmd/generic.nix
@@ -74,25 +74,16 @@ stdenv.mkDerivation (finalAttrs: {
   # https://issues.dlang.org/show_bug.cgi?id=19553
   hardeningDisable = [ "fortify" ];
 
-  patches =
-    lib.optionals (lib.versionOlder version "2.088.0") [
-      # Migrates D1-style operator overloads in DMD source, to allow building with
-      # a newer DMD
-      (fetchpatch {
-        url = "https://github.com/dlang/dmd/commit/c4d33e5eb46c123761ac501e8c52f33850483a8a.patch";
-        stripLen = 1;
-        extraPrefix = "dmd/";
-        hash = "sha256-N21mAPfaTo+zGCip4njejasraV5IsWVqlGR5eOdFZZE=";
-      })
-    ]
-    ++ [
-      (fetchpatch {
-        url = "https://github.com/dlang/dmd/commit/fdd25893e0ac04893d6eba8652903d499b7b0dfc.patch";
-        stripLen = 1;
-        extraPrefix = "dmd/";
-        hash = "sha256-Uccb8rBPBLAEPWbOYWgdR5xN3wJoIkKKhLGu58IK1sM=";
-      })
-    ];
+  patches = lib.optionals (lib.versionOlder version "2.088.0") [
+    # Migrates D1-style operator overloads in DMD source, to allow building with
+    # a newer DMD
+    (fetchpatch {
+      url = "https://github.com/dlang/dmd/commit/c4d33e5eb46c123761ac501e8c52f33850483a8a.patch";
+      stripLen = 1;
+      extraPrefix = "dmd/";
+      hash = "sha256-N21mAPfaTo+zGCip4njejasraV5IsWVqlGR5eOdFZZE=";
+    })
+  ];
 
   postPatch = ''
     patchShebangs dmd/compiler/test/{runnable,fail_compilation,compilable,tools}{,/extra-files}/*.sh
@@ -189,6 +180,7 @@ stdenv.mkDerivation (finalAttrs: {
 
     NIX_ENFORCE_PURITY= \
       make -C phobos unittest -j$checkJobs $checkFlags \
+        DISABLED_TESTS=std/datetime/timezone \
         DFLAGS="-version=TZDatabaseDir -version=LibcurlPath -J$PWD"
 
     runHook postCheck

--- a/pkgs/by-name/dm/dmd/package.nix
+++ b/pkgs/by-name/dm/dmd/package.nix
@@ -1,5 +1,5 @@
 import ./generic.nix {
-  version = "2.110.0";
-  dmdHash = "sha256-icXp9xWF2AI2gp7z/lQFAChmXfQePe9b5pbpQ9Mn19Y=";
-  phobosHash = "sha256-CmJpcHM+sIsaYBlpALCFoQFG+93s8gUyWmM0tYqjXkk=";
+  version = "2.112.1";
+  dmdHash = "sha256-UqNNUGoGmOV0lGNCGaFk8HbOE01qD55WuJ8NyQzqLJs=";
+  phobosHash = "sha256-CNvDSlerOXh6Qn060MNaSF1IIcbrcZzljyIxkm+TQQ0=";
 }

--- a/pkgs/by-name/dt/dtools/package.nix
+++ b/pkgs/by-name/dt/dtools/package.nix
@@ -8,13 +8,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "dtools";
-  version = "2.110.0";
+  version = "2.112.1";
 
   src = fetchFromGitHub {
     owner = "dlang";
     repo = "tools";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-xMEHnrstL5hAkhp8+/z1I2KZWZ7eztWZnUGLTKCfbBI=";
+    hash = "sha256-WHgV33fu1YhRQCERVatqXKEqmfHlUvZ4IXhga/yOQ3w=";
     name = "dtools";
   };
 

--- a/pkgs/by-name/du/dub/package.nix
+++ b/pkgs/by-name/du/dub/package.nix
@@ -10,7 +10,7 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "dub";
-  version = "1.39.0";
+  version = "1.41.0";
 
   enableParallelBuilding = true;
 
@@ -18,7 +18,7 @@ stdenv.mkDerivation (finalAttrs: {
     owner = "dlang";
     repo = "dub";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-73b15A9+hClD6IbuxTy9QZKpTKjUFYBuqGOclUyhrnM=";
+    hash = "sha256-86TzaRdu+CFM0Ld2fXxjaUUT9dl37MYPdmsvxE4ZUkE=";
   };
 
   postPatch = ''
@@ -139,6 +139,7 @@ stdenv.mkDerivation (finalAttrs: {
     rm -r test/pr2642-cache-db
     rm -r test/pr2644-describe-artifact-path
     rm -r test/pr2647-build-deep
+    rm -r test/issue2698-cimportpaths-broken-with-dmd-ldc
 
     ./test/run-unittest.sh
 

--- a/pkgs/by-name/ld/ldc/package.nix
+++ b/pkgs/by-name/ld/ldc/package.nix
@@ -32,13 +32,13 @@ in
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "ldc";
-  version = "1.41.0";
+  version = "1.42.0";
 
   src = fetchFromGitHub {
     owner = "ldc-developers";
     repo = "ldc";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-6LcpY3LSFK4KgEiGrFp/LONu5Vr+/+vI04wEEpF3s+s=";
+    hash = "sha256-adA14tx/bruGvHVoODz13f8h/U2ol1lK0ytxnypsLv8=";
     fetchSubmodules = true;
   };
 

